components/sharing-dialog/src/__tests__/SharingDialog.test.js

```diff
@@ -17,7 +18,9 @@ describe('SharingDialog widget', () => {
 
     const getSharingDialogWidget = props => {
```

Comment:
Sidenote: these abstractions make this test harder to work on in my opinion. Useful for app code, not as useful for tests, that are now all coupled and harder to understand and refactor. Same goes for the shareblock test above.